internal/sip/sip.go

```diff
@@ -55,7 +54,7 @@ func New(path string) (SIP, error) {
 	if err != nil {
 		return s, fmt.Errorf("SIP: New: %v", err)
 	}
-	if len(f) > 0 && strings.Contains(strings.ToLower(s.Path), "vecteur") {
+	if len(f) > 0 {
```

Comment:
Why are you removing this requirement?

Reply:
I removed it because it didn't seem important to me — If the SIP contains a "Prozess_Digitalisierung_PREMIS.xml" file then it must be a digitized SIP whether or not it has "vecteur" in the name. Conversely if a SIP has a "Prozess_Digitalisierung_PREMIS.xml" file but doesn't have "vecteur" in the same then it shouldn't be identified as "born digital" SIP in my opinion.

Reply:
@fiver-watson this was the requirement that checked for "vecteur" in the SIP name (that we made case insensitive). Is it okay to remove it? Not sure where it comes from.

Reply:
@fiver-watson I'm just waiting on your response to @jraddaoui before submitting this PR for a final code review.